testsuite/features/support/twopence_init.rb

```diff
@@ -232,7 +232,7 @@ def file_inject(node, local_file, remote_file)
 $product_version = product_version
 $use_salt_bundle = use_salt_bundle
 $pxeboot_mac = ENV['PXEBOOT_MAC']
-$pxeboot_image = ENV['PXEBOOT_IMAGE'] || 'sles15sp3o'
+$pxeboot_image = ENV['PXEBOOT_IMAGE'] || 'sles15sp4o'
```

Comment:
aaaand merge conflicts with Retail ... unless I merge my PR first :trollface:

Reply:
Yeah. I forgot about this. You missed that with your first PR and then added it later. Let`s see which PR will get merged before. I will always do a rebase before merging something.

Reply:
Me too. OK, ready for the fight :stuck_out_tongue: